main/main.cpp

```diff
@@ -391,8 +393,8 @@ void Main::print_help(const char *p_binary) {
 	OS::get_singleton()->print("                                               <path> should be absolute or relative to the project directory, and include the filename for the binary (e.g. 'builds/game.exe'). The target directory should exist.\n");
 	OS::get_singleton()->print("  --export-debug <preset> <path>               Same as --export, but using the debug template.\n");
 	OS::get_singleton()->print("  --export-pack <preset> <path>                Same as --export, but only export the game pack for the given preset. The <path> extension determines whether it will be in PCK or ZIP format.\n");
-	OS::get_singleton()->print("  --convert-3to4                               Converts project from Godot 3.x to Godot 4.x.\n");
-	OS::get_singleton()->print("  --validate-conversion-3to4                   Shows what elements will be renamed when converting project from Godot 3.x to Godot 4.x.\n");
+	OS::get_singleton()->print("  --convert-3to4 <max_file_kb> <max_line_size>              Converts project from Godot 3.x to Godot 4.x.\n");
+	OS::get_singleton()->print("  --validate-conversion-3to4 <max_file_kb> <max_line_size>  Shows what elements will be renamed when converting project from Godot 3.x to Godot 4.x.\n");
```

Comment:
```suggestion
	OS::get_singleton()->print("  --convert-3to4 [<max_file_kb> [<max_line_size>]]              Converts project from Godot 3.x to Godot 4.x.\n");
	OS::get_singleton()->print("  --validate-conversion-3to4 [<max_file_kb> [<max_line_size>]]  Shows what elements will be renamed when converting project from Godot 3.x to Godot 4.x.\n");
```
The `[ ]` syntax means that it's optional.